glimp/egl/display.cc

```diff
@@ -20,6 +20,7 @@
 #include <utility>
 #include <vector>
 
+#include "base/logging.h"
```

Comment:
Glimp does not depend on Chrome //base, so changes in //glimp should be undone. It's a lower layer standalone library - it's okay here to use Starboard directly.

Reply:
Done